lib/screens/chatscreen/chat_modules/chat_detail_page.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:shareacab/main.dart';
+import 'package:shareacab/screens/chatscreen/chat_components/chat_bubble.dart';
+import 'package:shareacab/screens/chatscreen/chat_components/chat_detail_page_appbar.dart';
+import 'package:shareacab/screens/chatscreen/chat_models/chat_message.dart';
+import 'package:shareacab/screens/chatscreen/chat_models/send_menu_items.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+enum MessageType{
+  Sender,
+  Receiver,
+}
+
+
+class ChatDetailPage extends StatefulWidget{
+  @override
+  _ChatDetailPageState createState() => _ChatDetailPageState();
+}
+
+class _ChatDetailPageState extends State<ChatDetailPage> {
+  List<ChatMessage> chatMessage = [
+    ChatMessage(message: 'Hi John', type: MessageType.Receiver),
+    ChatMessage(message: 'Hope you are doin good', type: MessageType.Receiver),
+    ChatMessage(message: "Hello Jane, I'm good what about you", type: MessageType.Sender),
+    ChatMessage(message: "I'm fine, Working from home", type: MessageType.Receiver),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+    ChatMessage(message: 'Oh! Nice. Same here man', type: MessageType.Sender),
+  ];
+
+  List<SendMenuItems> menuItems = [
+    SendMenuItems(text: 'Photos & Videos', icons: Icons.image, color: Colors.amber),
+    SendMenuItems(text: 'Document', icons: Icons.insert_drive_file, color: Colors.blue),
+    SendMenuItems(text: 'Audio', icons: Icons.music_note, color: Colors.orange),
+    SendMenuItems(text: 'Location', icons: Icons.location_on, color: Colors.green),
+    SendMenuItems(text: 'Contact', icons: Icons.person, color: Colors.purple),
```

Comment:
Define these colors in main.dart. I guess use whatever names for now. @tmibvishal can guide on that

Reply:
Make a function in main.dart and pass an int i in it to get that color for menu